common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -210,12 +210,14 @@ def student_view(self, context):
         self._capture_basic_metrics()
 
         # Is this sequential part of a timed or proctored exam?
+        masquerading = context.get('specific_masquerade', False)
+        view_html = None
```

Comment:
ah. didn't know that.  sounds good.

("special" is such a generic term.  But since there's precedence in using it with the feature flag - I'm on board for now.)